src/test/integration/mocks/server.ts

```diff
@@ -4,34 +4,31 @@
  */
 
 import { ZodiosPathsByMethod, ZodiosResponseByPath } from '@zodios/core/lib/zodios.types';
-import { ResponseResolver, rest, RestContext, RestRequest } from 'msw';
+import { ResponseResolver, http, HttpResponse } from 'msw';
 import { setupServer } from 'msw/node';
 import { TerraformCloudAPIUrl, apiClient } from '../../../terraformCloud';
 
 type Api = typeof apiClient.api;
 
 export function mockGet<Path extends ZodiosPathsByMethod<Api, 'get'>>(
   path: Path,
-  resolver: ResponseResolver<RestRequest, RestContext, Awaited<ZodiosResponseByPath<Api, 'get', Path>>>,
+  resolver: ResponseResolver<{}, never, Awaited<ZodiosResponseByPath<Api, 'get', Path>>>,
```

Comment:
The `ResponseResolver` type generic has changed:

- The first argument is now an `Extras` object. In your context, we can leave it empty.
- The second argument is the request body type. Since you are intercepting a GET request here, its body type is always `never`.
- The third argument is the response body type. I left your code that extracts the body type from the API client as-is.